controllers/csm_controller.go

```diff
@@ -1334,6 +1334,11 @@ func (r *ContainerStorageModuleReconciler) PreChecks(ctx context.Context, cr *cs
 		if err != nil {
 			return fmt.Errorf("failed powerflex validation: %v", err)
 		}
+		// zoning initially applies only to pflex
+		err = r.ZoneValidation(ctx, cr, namespace)
```

Comment:
Do we need to pass in namespace here and in pre-checks?
If we have the cr, we should be able to get the namespace from 
cr.Namespace.
Like here: https://github.com/dell/csm-operator/blob/f54c83af80cc5d3428019ff6b78e36c4260dab17/controllers/csm_controller.go#L1371

Reply:
makes sense. updated function.